### ISSUE TITLE
Does this work with Kinetic?

### ISSUE THREAD

**Issue by hjjayakrishnan** [ccny-ros-pkg/phidgets_drivers#31]

**Comment by mintar** [comment]
Yes, it does. This repo has moved to https://github.com/ros-drivers/phidgets_drivers . Simply use the `kinetic` branch there. The package has also been released, so you can simply type `sudo apt-get install ros-kinetic-phidgets-drivers`.

@muhrix This issue can be closed.

**Comment by ShanLuo** [comment]
Hi mintar, I installed the phidgets drivers by `sudo apt-get install ros-kinetic-phidgets-drivers`, but when I run `rosrun phidgets_imu phidgets_imu_node`, it says "Couldn't find executable named phidgets_imu_node below /opt/ros/kinetic/share/phidgets_imu". I am using ROS kinetic. Any ideas?

**Comment by ShanLuo** [comment]
And when I run "roslaunch phidgets_imu imu.launch", I got

Failed to load nodelet [/ImuFilterNodelet] of type [imu_filter_madgwick/ImuFilterNodelet] even after refreshing the cache: According to the loaded plugin descriptions the class imu_filter_madgwick/ImuFilterNodelet with base class type nodelet::Nodelet does not exist. Declared types are  phidgets_imu/PhidgetsImuNodelet

**Comment by hjjayakrishnan** [comment]
@ShanLuo Try to cd into the launch directory and then run `roslaunch imu.launch` Let me know what happens.

**Comment by ShanLuo** [comment]
Hi  hjjayakrishnan, thanks for your comments. The problem has been solved by installing `imu_tools`.
http://wiki.ros.org/imu_tools?distro=kinetic

**Comment by mintar** [comment]
Hi @ShanLuo , great that you could solve your problem. BTW, this repo is deprecated! It has moved to https://github.com/ros-drivers/phidgets_drivers, and that repo has a kinetic branch.
